script/ozw_emulator.py

```diff
@@ -0,0 +1,95 @@
+"""Emulate MQTT Broker with OZW Daemon instance from MQTT dump."""
+
+# Experimental ! For debugging purposes
+# This will host a MQTT (3.1.1) Broker on localhost:1883
+# Content of the provided MQTT dump file will be published on the broker
+# Setvalue command will be handled too.
+# Connect with Hass + Z-Wave MQTT Addon and/or MQTT Explorer to test userdumps
+
+# WARNING: Use dev version of HBMQTT pip install git+git://github.com/beerfactory/hbmqtt
+# Use manual yaml config for mqtt in Hass, because Config flow seems to default to 3.1
+
+import argparse
+import asyncio
+import json
+import logging
+
+from hbmqtt.broker import Broker
+from hbmqtt.client import MQTTClient
+from hbmqtt.mqtt.constants import QOS_0
+
+BROKER_CONFIG = {
+    "listeners": {
+        "default": {"max-connections": 50000, "type": "tcp"},
+        "my-tcp-1": {"bind": "127.0.0.1:1883"},
+    },
+    "timeout-disconnect-delay": 2,
+    "auth": {"allow-anonymous": "true", "plugins": ["auth.anonymous"]},
+    "topic-check": {"enabled": True, "plugins": ["topic_taboo"]},
+}
+
+
+def get_args():
+    """Get arguments."""
+    parser = argparse.ArgumentParser(description="OZW Emulator")
+    parser.add_argument("filename", type=str, help="File with dump from mqtt.")
+    return parser.parse_args()
+
+
+async def publish_from_file():
+    """Publish all topics from the provided dump file."""
+    with open(dump_file, "rt") as fp:
+        for line in fp:
+            topic, payload = line.strip().split(",", 1)
+            await mqtt_client.publish(topic, payload.encode(), retain=True)
+
+
+async def process_messages():
```

Comment:
Maybe accept the client here to not depend on the global scope.

Reply:
done